### ISSUE TITLE
Multiple "Warning: Require cycle" messages

### ISSUE THREAD

**Issue by Javi** [Instabug/Instabug-React-Native#406]
# Steps to Reproduce the Problem
Simply running a build with Instabug enabled

### Expected Behavior
No warnings at all

### Actual Behavior
Several of these appear
![image](https://user-images.githubusercontent.com/4081895/65081248-8e1c4b80-d979-11e9-8b3d-4c2e03fd9540.png)
![image](https://user-images.githubusercontent.com/4081895/65081275-9ffdee80-d979-11e9-8a50-ebc8821aed3e.png)

### SDK Version
^8.6.4

### React Native, iOS and Android Versions
RN 0.60.5

**Comment by alyezz** [comment]
Thanks for the report. We're on it.

**Comment by saudahmed** [comment]
@Javi which steps did you take to run react-native 0.60.5 with instabug.

The documentation are pretty confusing as I am not even able to integrate simply the SDK

**Comment by Javi** [comment]
I called this on the JS side
```
Instabug.startWithToken("KEY", [
    Instabug.invocationEvent.shake
]);
```
and this on the android side on the `onCreate` method
```
new RNInstabugReactnativePackage.Builder("KEY", MainApplication.this)
                .setInvocationEvent("shake")
                .setPrimaryColor("#1D82DC")
                .setFloatingEdge("left")
                .setFloatingButtonOffsetFromTop(250)
                .build();
```

**Comment by alyezz** [comment]
@Javi we just merged a PR that includes a fix for this #410 and will be included in our next release.
